apps/src/lib/node/ledger/protocol/mod.rs

```diff
@@ -85,8 +84,7 @@ where
         Self {
             block_gas_meter: &mut shell.gas_meter,
             write_log: &mut shell.write_log,
-            storage: &mut shell.storage,
-            wasm_dir: shell.wasm_dir.as_path(),
+            storage: &shell.storage,
```

Comment:
We were passing here `&mut shell.storage` before though the `storage` field itself was `&'a Storage<D, H>` so I think the `mut` here wasn't doing anything?